src/docs.js

```diff
@@ -6,18 +6,25 @@ let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
 let zlib = require('zlib');
 let rootdir = require('app-root-dir');
+let aws = require('aws-sdk');
+let client = require('taskcluster-client');
+let S3UploadStream = require('s3-upload-stream');
+let debug = require('debug')('taskcluster-lib-docs');
 
 async function documenter(options) {
   options = _.defaults({}, options, {
+    credentials: {},
+    project: rootdir.get() + '/package.json',
     tier: null,
     schemas: {},
     menuIndex: 10,
     docsFolder: rootdir.get() + '/docs',
     references: [],
+    publish: process.env.NODE_ENV == 'production',
   });
 
+  assert(options.project, 'options.project must be given');
```

Comment:
Let's remove this as a requirement. Rather, if it's null, set it as follows:

```js
if (!options.name) {
  let pack = require(rootdir.get() + '/package.json');
  options.project = pack.name;
}
```

Or something like that.